src/_pytest/helpconfig.py

```diff
@@ -191,9 +193,12 @@ def showhelp(config: Config) -> None:
             tw.write(" " * (indent_len - spec_len - 2))
             wrapped = textwrap.wrap(help, columns - indent_len, break_on_hyphens=False)
 
-            tw.line(wrapped[0])
-            for line in wrapped[1:]:
-                tw.line(indent + line)
+            if not wrapped:
+                tw.write(" " * (indent_len - spec_len - 2))
```

Comment:
Can you explain why this line is necessary? From what I can see it will be better to omit it (I think it just adds trailing whitespace).

Reply:
To be honest, I added it for the whole help to display correctly, but testing without shows this is, as you say, trailing whitespace. I've removed it and updated the PR!